File: CHALLENGE_02/C#/MariioM.cs
using System;

namespace Challenge_02
{
    internal class Program
    {
        static void Main(string[] args)
        {
            const string input = "&###@&*&###@@##@##&######@@#####@#@#@#@##@@@@@@@@@@@@@@@*&&@@@@@@@@@####@@@@@@@@@#########&#&##@@##@@##@@##@@##@@##@@##@@##@@##@@##@@##@@##@@##@@##@@##@@&";
            char[] inputList = input.ToCharArray();
            int result = 0;
            foreach (char symbol in inputList)
            {
                switch(symbol)
                {
                    case '#':
                        result++;break;
                    case '@':
                        result--; break;
                    case '*':
                        result *= result; break;
                    case '&':
                        Console.Write(result); break;
                    default: continue;
                }
            }
            Console.ReadLine();
        }
    }
}

File: CHALLENGE_02/Kotlin/MariioM.kt
fun main(){
    val input = "&###@&*&###@@##@##&######@@#####@#@#@#@##@@@@@@@@@@@@@@@*&&@@@@@@@@@####@@@@@@@@@#########&#&##@@##@@##@@##@@##@@##@@##@@##@@##@@##@@##@@##@@##@@##@@##@@&"
    val inputList = input.toCharArray()
    var result = 0
    for(symbol in inputList){
       when(symbol){
            '#' -> result++
            '@' -> result--
            '*' -> result *= result
            '&' -> print(result)
            else -> continue
        }
    }
}